docs/rst/notes/forthcoming_version.rst

```diff
@@ -28,9 +28,11 @@ Next release will include the following **DDS Recorder tool configuration featur
 * New configuration options (``timestamp-format`` and ``local-timestamp``) available for :ref:`output file <recorder_usage_configuration_outputfile>` settings.
 * New configuration option (``topics``) to configure the :ref:`Manual Topics <recorder_manual_topics>`.
 * Rename ``max-reception-rate`` to ``max-rx-rate``.
+* Record information in either ROS 2 format or the raw DDS format (see :ref:`Topic Type Format <recorder_usage_configuration_topictypeformat>`).
 
 Next release will include the following **DDS Replayer tool configuration features**:
 
 * New configuration option (``topics``) to configure the :ref:`Manual Topics <replayer_manual_topics>`.
 * New configuration option (``max-tx-rate``) to configure the :ref:`Max transmission rate <replayer_max_tx_rate>`.
 * Remove the support for `Built-in Topics <https://dds-recorder.readthedocs.io/en/v0.2.0/rst/replaying/usage/configuration.html#built-in-topics>`_.
+* Replay information in either ROS 2 format or the raw DDS format.
```

Comment:
```suggestion
* Replay data in either ROS 2 format or the raw DDS format.
```

Reply:
I am a bit confused about this, because the replayer cannot be configured to behave in the two modes. The replayer always replays raw DDS data right? I mean, if ros_types was false in recorder nothing is transformed in the replayer, but if ros_types was true and some types were demangled by the recorder, the replayer would always "remangle" it so what is sent is the original raw DDS data. Am I wrong?

Reply:
Yes, you are correct.